DirectProgramming/DPC++FPGA/ReferenceDesigns/anr/src/mp_math.hpp

```diff
@@ -0,0 +1,157 @@
+#ifndef __MP_MATH__
+#define __MP_MATH__
+

```

Comment:
Please add a description of the purpose / contents of this file in a comment at the top of the file